### ISSUE TITLE
Disable jump to message functionality on Quoter

### ISSUE THREAD

**Issue by PointyDev** [samogot/betterdiscord-plugins#3]
By default, the plugin would add a link as the author URL that redirects to an info page if you don't have the plugin installed. This isn't optimal as some people disprove of client mods and I'd rather them not be redirected to a page like that.

**Comment by samogot** [comment]
1. You can't choose who would see links in you messages with quotes. So if there is a link, everyone without plugin installed would see it.
2. There is no other way to provide information about quoted message id, to allow jumping to that message on click. We should have any link, with that info coded into it.
3. In my opinion, link with info about plugin is better then some random or broken link - as it will be somehow helpful even for people without plugin.
4. I won't provide an option to don't post links at all, therefore don't use jumps on click. Such option will affect other plugin users, who may expect to jump from your quote, and if there is no link - they can't
5. Technically I can add an option to use some other link, or even broken link, instead of plugin info, but would it be really better for you?

**Comment by PointyDev** [comment]
I'm a bug hunter on [Discord Testers](https://discord.gg/discord-testers), and using client mods is frowned upon. I just think people would like an option to turn this functionality off, in the meantime I've just removed one line of code so that it's disabled for me.

**Comment by samogot** [comment]
Yeah, it's easy to disable posting links, but you also disbled jumping from your quotes, both for you and for other plugin users. Theoretically, they may even come here and create a bug about they can't jum from your quotes :)
I care about it so much, because jumps feature was my main reason to develop this plugin :)

**Comment by PointyDev** [comment]
And my main reason for using the plugin is because I wanted to be able to quote message, without some proprietary page telling people to download your plugin if they accidentally click the link without the plugin installed.

**Comment by shinji257** [comment]
While I see the reason why you want it @PointyDev isn't the point of Discord Testers to test the pre-release version for bugs from the stock code?  Adding mods (even to just quote) can poison the report and may make it invalid.
